platform/resource-bundle-locator/src/main/resources/SourceKeywordsBundle.properties

```diff
@@ -0,0 +1,5 @@
+Data\ Source=Data Source
```

Comment:
✏️ This seems to work but the contention I have usually seen is to use camel cased keys or keys more like our metacard attributes.  I also think it would be useful to include context about the value instead of just using the English form for all keys.  For example, `Source`, `SourcePlural`, and `SourcePluralLowerCase`.
https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#toLowerCase-java.util.Locale-

Reply:
This is a specific case where we can use camel case for this matter. There is a place in the UI where I perform a conditional match for registered keywords.

`const menuText = properties.keywords[this.options.text] || this.options.text`

So when we made need to perform a conditional match against a phrase that contains spaces we'll need to write the keywords like `Data\ Source=Data Source`

Reply:
Typically, entries in a resource bundles are defined using a dot notation like `data.source`

Reply:
I've changed them to use a dot notation. This file also needed to be deleted as it was no longer being used